pandas/core/indexes/datetimelike.py

```diff
@@ -30,15 +31,20 @@
 _index_doc_kwargs = dict(ibase._index_doc_kwargs)
 
 
-class DatetimeIndexOpsMixin(DatetimeLikeArrayMixin):
+def ea_passthrough(name):
```

Comment:
Some variant of it, or it could be rolled in to the Delegators (though that might then need to be moved to the top of the file).  Regardless, I'll add a docstring.